test/fixtures/ledger_transaction_fixtures.js

```diff
@@ -219,12 +241,47 @@ module.exports = {
     }
 
     return {
-      getPactified: () => {
-        return pactRegister.pactify(data)
-      },
-      getPlain: () => {
-        return data
+      getPactified: () => pactRegister.pactify(data),
+      getPlain: () => data
+    }
+  },
+  validTransactionEventsResponse: (opts = {}) => {
+    const data = {
+      transaction_id: opts.transaction_id || 'ht439nfg2l1e303k0dmifrn4fc',
+      events: (opts.payment_states) ? buildPaymentEvents(opts) : buildChargeEventStateWithDefaults(opts)
```

Comment:
Is it correct?
`buildChargeEventStateWithDefaults` returns a state object, not an array of events.